esp/applications/eclwatch/ldap_authorization_map.yaml

```diff
@@ -0,0 +1,153 @@
+         -  path: FileSprayAccess
+            resource: FileSprayAccess
+            description: Access to spraying files
+         -  path: FileDesprayAccess
+            resource: FileDesprayAccess
+            description: Access to despraying of files
+         -  path: FileDkcAccess
+            resource: FileDkcAccess
+            description: Access to dkcing of key files
+         -  path: FileIOAccess
+            resource: FileIOAccess
+            description: Access to files in dropzone
+         -  path: FileUploadAccess
+            resource: FileUploadAccess
+            description: Access to upload files to LZ
+         -  path: WsEclAccess
+            resource: WsEclAccess
+            description: Access to WS ECL service
+         -  path: RoxieQueryAccess
+            resource: RoxieQueryAccess
+            description: Access to Roxie queries and files
+         -  path: ClusterTopologyAccess
+            resource: ClusterTopologyAccess
+            description: Access to cluster topology
+         -  path: OwnWorkunitsAccess
+            resource: OwnWorkunitsAccess
+            description: Access to own workunits
+         -  path: OthersWorkunitsAccess
+            resource: OthersWorkunitsAccess
+            description: Access to others' workunits
+         -  path: EclDirectAccess
+            resource: EclDirectAccess
+            description: Access to ECL direct service
+      WsWorkunits:
+         Feature:
+         -  path: OwnWorkunitsAccess
+            resource: OwnWorkunitsAccess
+            description: Access to own workunits
+         -  path: OthersWorkunitsAccess
+            resource: OthersWorkunitsAccess
+            description: Access to others' workunits
+      WsTopology:
+         Feature:
+         -  path: ClusterTopologyAccess
+            resource: ClusterTopologyAccess
+            description: Access to cluster topology
+         -  path: MachineInfoAccess
+            resource: MachineInfoAccess
+            description: Access to machine information
+      WsDfu:
+         Feature:
+         -  path: DfuAccess
+            resource: DfuAccess
+            description: Access to DFU
+      WsDfuXRef:
+         Feature:
+         -  path: DfuXrefAccess
+            resource: DfuXrefAccess
+            description: Access to DFU XRef
+      FileSpray:
+         Feature:
+         -  path: DfuWorkunitsAccess
+            resource: DfuWorkunitsAccess
+            description: Access to DFU workunits
+         -  path: DfuExceptionsAccess
+            resource: DfuExceptions
+            description: Access to DFU exceptions
+         -  path: FileSprayAccess
+            resource: FileSprayAccess
+            description: Access to spraying files
+         -  path: FileDesprayAccess
+            resource: FileDesprayAccess
+            description: Access to despraying of files
+         -  path: FileDkcAccess
+            resource: FileDkcAccess
```

Comment:
Do we still support DKC ??

Reply:
I don't know.  Once this is final I'll open up a JIRA for reviewing things that might be removed because they are deprecated, etc.